src/main/java/io/github/syst3ms/skriptparser/expressions/ExprAnswer.java

```diff
@@ -0,0 +1,60 @@
+package io.github.syst3ms.skriptparser.expressions;
+
+import io.github.syst3ms.skriptparser.Parser;
+import io.github.syst3ms.skriptparser.effects.EffAsk;
+import io.github.syst3ms.skriptparser.lang.Expression;
+import io.github.syst3ms.skriptparser.lang.TriggerContext;
+import io.github.syst3ms.skriptparser.parsing.ParseContext;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Get back the last entered answer in a {@link EffAsk} effect.
+ * This will return the last line of the console.
+ *
+ * @name Last Answer
+ * @type EXPRESSION
+ * @pattern [the] [last] (answer|response)
+ * @since ALPHA
+ * @author ItsTheSky
+ */
+public class ExprAnswer implements Expression<String> {
+
+    static {
+        Parser.getMainRegistration().addExpression(
+                ExprAnswer.class,
+                String.class,
+                true,
+                "[the] [last] (answer|response)"
+        );
+        ANSWERS = new HashMap<>();
+    }
+
+    private static final Map<TriggerContext, String> ANSWERS;
```

Comment:
Honestly, I feel that this map is unnecessary? I think there should just be one String static field that holds the latest answer for all triggers. We shouldn't really make a difference between contexts. Not sure what your opinion is on this.

Reply:
I first through about that, but if multiple context are running they could have interference ig? 
Anyways the effect wait before running the next trigger, so you've surly right 😂

Reply:
You're right. You always want the latest answer that you asked for in this specific trigger. There would probably be no use-case where my suggestion would be beneficial.